database/Room.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+
+const roomSchema = new mongoose.Schema({
+  id: {type: Number, min: 0, max: 99},
+  name: String,
+  reviews: [
+    {
+      reviewee: String,
+      revieweeAvatar: String,
+      createdAt: Date,
+      text: String
+    }
+  ],
+  rating: {
+    overall: {type: Number, min: 1, max: 5},
+    accuracy: {type: Number, min: 1, max: 5},
+    location: {type: Number, min: 1, max: 5},
+    cleanliness: {type: Number, min: 1, max: 5},
+    communication: {type: Number, min: 1, max: 5},
+    checkIn: {type: Number, min: 1, max: 5},
+    value: {type: Number, min: 1, max: 5}
+  }
+})
+
+const roomsModel = mongoose.model('rooms', roomSchema);
```

Comment:
See [https://mongoosejs.com/docs/models.html#compiling](https://mongoosejs.com/docs/models.html#compiling)

It seems like this already works for you but mongoose docs specify that the first argument of mongoose.model is a singular noun.

Reply:
You're right. Changing now.